doc/classes/SpinBox.xml

```diff
@@ -5,6 +5,19 @@
 	</brief_description>
 	<description>
 		SpinBox is a numerical input text field. It allows entering integers and floats.
+		[b]Example:[/b]
+		[codeblock]
+		func _ready():
+		  var spin_box = SpinBox.new()
```

Comment:
Indentation should be done with 4 spaces, not 2. But if you remove the `func _ready():`, then no indentation is needed.